src/components/typography/text/text.js

```diff
@@ -63,9 +80,10 @@ Subheadline.propTypes = {
     'positive',
     'negative',
   ]),
-  children: PropTypes.node.isRequired,
   title: nonEmptyString,
   truncate: PropTypes.bool,
+  intlMessage: intlMessageShape,
```

Comment:
Should this be flipped to the case below. Given there are no children then this?

Reply:
Ah thanks, good catch!